src/models/identifier_count.ts

```diff
@@ -0,0 +1,26 @@
+import mongoose, { Schema, model } from "mongoose";
+import type { Document, Model } from "mongoose";
+
+interface InterfaceIdentifier extends Document{
+  id: string,
+  sequence_value : number
+}
+
+const identifierSchema = new Schema<InterfaceIdentifier>({
+  _id: { type: String, required: true },
+  sequence_value: { type: Number },
+});
+
+let lastIdentifier:Model<InterfaceIdentifier>
+if (mongoose.models.identifier_count) {
+  lastIdentifier = model("identifier_count");
+}
+else {
+  lastIdentifier = model("identifier_count",identifierSchema)
+}
+
+
+export const identifier_count = lastIdentifier;
+
+
+  
```

Comment:
don't understand what's being done here, explain please

Reply:
This looks like some wrapper function to me @xoldyckk